runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java

```diff
@@ -485,7 +486,15 @@ public void layoutContainer(Container parent)
 				null);
 
 			titleToolbar.addComponent(sidebarNavigationButton, sidebarNavigationJButton);
-			toggleSidebar();
+
+			// Close sidebar if config is set for it.
+
+			if (configManager.getConfiguration(CONFIG_GROUP, CONFIG_CLIENT_SIDEBAR_CLOSED) == null)
```

Comment:
this is just wrong

Reply:
In hindsight seems maybe you meant to remove the config related code entirely, even the remembered state? Anyway seems the travis build failed due to some env variable issue now, which seems to be out of my control.

Reply:
Realized the build failed due to another checkstyle issue, I got confused because travis said "no environment variables set"